src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/model/VisitFilter.kt

```diff
@@ -5,12 +5,8 @@ import java.time.LocalDateTime
 data class VisitFilter(
   val prisonerId: String? = null,
   val prisonCode: String? = null,
```

Comment:
should we change this to a non optional field and startdatetime and enddatetime?